### PR TITLE
Fix small errors in the logs sub-module:

### PR DESCRIPTION
**Pull request by mohierf**
- erroneous logs
- check if configured logs collection exist to confirm correct connection

### CODE REVIEW

File: module/submodules/logs.py [hunk 1]
```diff
@@ -218,7 +217,7 @@ def get_ui_availability(self, elt, range_start=None, range_end=None):
                 if '_id' in log:
                     del log['_id']
                 records.append(log)
-            logger.debug("[mongo-logs] %d records fetched from database.", records.count())
+            logger.debug("[mongo-logs] %d records fetched from database.", len(records))
```
Comment (maethor) [inline module/submodules/logs.py:220]: I don't know about this. As I understand it, `records` is a mongo cursor that behaves like an iterator. `len(records)` will force it to be convertir as a list, while `records.count()` use a mongo method to compute the size of the iterator, wich may be way faster.

**Comment by maethor** [comment]
@mohierf I left you a comment in the code.

File: module/submodules/logs.py (hunk 1 above)
Comment (mohierf) [reply module/submodules/logs.py:220]: @maethor: in this place, records is a list not a pymongo Cursor ... as such you cannot use the `count` method and you must use `len` ...

Comment (maethor) [reply module/submodules/logs.py:220]: OK then :)

**Pull request merged**